src/main/haskell/kore/src/Kore/ASTPrettyPrint.hs

```diff
@@ -738,8 +738,8 @@ instance (MetaOrObject level, PrettyPrint (variable level))
 
 -- TODO: when refactoring these, consider removing `writeTwoFieldStruct`
 instance MetaOrObject level => PrettyPrint (UnificationError level) where
-    prettyPrint flags (ConstructorClash h1 h2) =
-        writeTwoFieldStruct flags "ConstructorClash" h1 h2
+    prettyPrint flags (PatternClash h1 h2) =
+        writeTwoFieldStruct flags "PatternClash" h1 h2
```

Comment:
Maybe. @virgil-serbanuta what have you to say about it?  The main reason for it was to satisfy the `--pedantic` build option.

Reply:
Since we didn't delete this, we still have to maintain it.

I don't remember exactly what happened with `Pretty` vs this printer, but in order to delete it, we should use the `Pretty` stuff in the regression tests and we should see that the output seems reasonable (i.e. pretty or as readable as our term representation can be).